luthier/config/src/tables.lua

```diff
@@ -136,19 +136,50 @@ M.physical = {
     Logical tables are defined in table M.logical. They map logical table
     names to their configuration:
 
-    * name - The name of the table
+    * The table is keyed on the name of the logical table
+    * category - the category in which the logical table belongs.
+        Defaults to 'GENERAL'
+    * longName - a longer name for descriptive uses.
         Defaults to the configuration's key
     * description - Brief documentation about the logical table.
         Defaults to the configuration's key
-    * metrics - A set of API metrics' name for this logical table, these 
-        should be a subset of the metrics configured in metrics.lua.
+    * type - [Mandatory] The nature of this logical table, used to build the table internally.
+        Currently supports:
+            * default - A simple set of rules just enough to describe a logical table correctly.
+                if 'default' is selected, the following fields will be used in the config:
+                    * metrics
+                    * dimensions
+                    * granularities
+                    * physicalTables
+                    * dateTimeZone
+                    * category
+                    * longName
+                    * retention
+                    * description
+                the following fields will be ignored:
+                    * apiFilters
+            * comprehensive - a list of rules to fully configure the logicalTable. This is not implemented yet.
+        Defaults to 'default'.
+    * metrics - A set of API metrics' name for this logical table, these
+        should be a subset of the metrics configured in metrics.lua;
+        Defaults to an empty lua table.
     * dimensions - A set of dimensions for this logical table, the set of
         dimensions should be a subset of dimensions in its dependent physical
         tables.
-    * granularity - A group of available granularities of this logical table, the
+        Defaults to an empty lua table.
+    * dateTimeZone - A case sensitive name according to joda's dateTimeZone to indicate the
+        which zone this physical table belongs in. See further:
+        https://www.joda.org/joda-time/timezones.html
+        Defaults to "UTC"
+    * retention - a String in the ISO 8601 Duration format, i.e. PnYnMnDTnHnMnS
```

Comment:
In addition to the format of the data, we should also speckiy what the data means. For example `metadata
meant to tell external customers the retention policy for the underlying druid table(s). Not used
internally by Fili.`

The same applies to `dateTimeZone`.

Reply:
added them in.